h/schemas/annotation.py

```diff
@@ -160,9 +163,9 @@ def validate(self, data):
             new_appstruct["shared"] = False
 
         if "target" in appstruct:
-            new_appstruct["target_selectors"] = _target_selectors(
-                appstruct.pop("target")
-            )
+            targets = appstruct.pop("target")
+            new_appstruct["target_description"] = _target_description(targets)
+            new_appstruct["target_selectors"] = _target_selectors(targets)
```

Comment:
Pop `target[0].description` out into a top-level `target_description` field for storage in the new `annotation.target_description` column, the same as `target[0].selectors` is popped out into the top-level `annotation.target_selectors` column.